WorldsAdriftServer/WorldsAdriftServer.cs

```diff
@@ -7,8 +11,23 @@ internal class WorldsAdriftServer
     {
         static void Main( string[] args )
         {
-            int restPort = 8080;
 
+            if (File.Exists("Config.ini"))
+            {
+                Console.WriteLine("The Config File Already exist");
```

Comment:
I will modify a bit later, maybe adding a parameter for "debug mode" which allow output of logs for server name / port / config error / spawn rate etc.... and I'm looking to add a check to see if the config file is valid and if not renaming the current config with "nv_config.ini" (not valid_config.ini) and creating a default / blank config file to let the server launch. I will probably need some feedback on this and the backup config name because I'm not fully confident about the naming.